config/crd/bases/instancemgr.keikoproj.io_instancegroups.yaml

```diff
@@ -3,8 +3,6 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  annotations:
-    controller-gen.kubebuilder.io/version: v0.2.4
```

Comment:
Can you update to controller-gen 0.2.4 (go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.4) and then run `make manifests`, you are using an older controller-gen which tries to add back all this stuff :)

Reply:
done

Reply:
after running `make manifests` it should modify these files back (CRD + role.yaml) to the minimal version, which you can check in to revert these changes